### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by annaeliana7**
In this feature, I :

- Added Webpack
- Implemented toDO list
- Added README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,141 @@
...
-> This project is to test linters
-## Built With
-- HTML and CSS
+<a name="readme-top"></a>
 
-## Getting Started
+<div align="center">
+
+  <img src="kefi-Unearthed-5" alt="logo" width="140"  height="auto" />
+  <br/>
```
Comment (Gambit142) [inline README.md]: - [ ] The link to your logo is broken. Kindly fix it so that your _**README.md**_ file looks professional. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @annaeliana7 👋, 

Good job so far!

### Highlights
- All linters checks passed ✔️ 
- Webpack was used to bundle JavaSCript files ✔️ 
- Renders static todo list in `DOM` ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (annaeliana7) [reply README.md]: Thank you for your change request. The README has been corrected.

Comment (Whoistolu) [reply README.md]: Well resolved!

**Review by Whoistolu — APPROVED** [review]
### Hi @annaeliana7 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**